superdesk/data_updates.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+import superdesk.metadata.utils
+import datetime
+import superdesk
+
+
+class DataUpdatesResource(Resource):
```

Comment:
you could set this as `internal_resource` so there are no http endpoints created, it doesn't need it right?